http/middleware_test.go

```diff
@@ -134,6 +165,61 @@ func TestWithHeaderAuthorization_Caching(t *testing.T) {
 	require.Equal(t, 2, totalJWKSRequests)
 }
 
+func TestWithHeaderAuthorization_CustomFailureHandler(t *testing.T) {
+	kid := "kid-" + t.Name()
+	// Mock the Clerk API server. We expect requests to GET /jwks.
+	clerkAPI := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path == "/jwks" && r.Method == http.MethodGet {
```

Comment:
❓ Same question as above regarding the JWKS endpoint path.

Reply:
As discussed above, see https://github.com/clerk/clerk-sdk-go/pull/276 for details.